Bottom: 22cbc710d582344776366e23fe8de23ad311868b
Top:    ec49587cfd431d0d6e6c8fd76032dc9ede0e14bf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:09 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7039457..55571cf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1251,8 +1251,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
